pkg/log/log.go

```diff
@@ -14,13 +14,15 @@ var Logger logr.Logger
 
 func init() {
 	// Build a zap development logger.
-	zapLog, err := zap.NewDevelopment()
+	zapLogger, err := zap.NewDevelopment(zap.AddCallerSkip(1))
 	if err != nil {
 		panic(fmt.Sprintf("error building logger: %v", err))
 	}
+	defer zapLogger.Sync()
```

Comment:
Food for thought: deferred calls aren't honored if `main.main` exits, right? Seems like another reason to manage logging setup/cleanup closer to the entry point and outside `init()` (to increase the odds of successfully flushing prior to a crash)